frontend/packages/dev-console/integration-tests/tests/serverless.scenario.ts

```diff
@@ -0,0 +1,148 @@
+import * as catalogPageView from '../../../../integration-tests/views/catalog-page.view';
+import * as operatorView from '../../../operator-lifecycle-manager/integration-tests/views/operator.view';
+import * as operatorHubView from '../../../operator-lifecycle-manager/integration-tests/views/operator-hub.view';
+
+const JASMINE_DEFAULT_TIMEOUT_INTERVAL = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+const JASMINE_EXTENDED_TIMEOUT_INTERVAL = 2000 * 60 * 5;
+
+describe('Serverless', async () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/k8s/cluster/projects`);
+    await switchPerspective(Perspective.Administrator);
+    expect(sideHeader.getText()).toContain('Administrator');
+
+    // Extend the default jasmine timeout interval just in case it takes a while for the htpasswd idp to be ready
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_EXTENDED_TIMEOUT_INTERVAL;
+  });
+
+  afterAll(() => {
+    // Set jasmine timeout interval back to the original value after these tests are done
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_DEFAULT_TIMEOUT_INTERVAL;
+  });
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  it('displays subscription creation form for selected Operator', async () => {
+    await crudView.isLoaded();
+    await browser.wait(until.presenceOf(sidenavView.navSectionFor('Operators')));
+    await sidenavView.clickNavLink(['Operators', 'OperatorHub']);
+    await crudView.isLoaded();
+    await catalogView.categoryTabsPresent();
+    await catalogView.categoryTabs.get(0).click();
+    await catalogPageView.catalogTileFor('OpenShift Serverless Operator').click();
+    await browser.wait(until.visibilityOf(installServerlessOperator));
+    await installServerlessOperator.element(by.linkText('Install')).click();
+    await operatorHubView.createSubscriptionFormLoaded();
+    expect(operatorHubView.createSubscriptionFormName.getText()).toEqual(
+      'OpenShift Serverless Operator',
+    );
+    await browser.wait(until.visibilityOf(operatorHubView.createSubscriptionFormInstallMode));
+    await operatorHubView.allNamespacesInstallMode.click();
+
+    expect(operatorHubView.createSubscriptionError.isPresent()).toBe(false);
+    expect(operatorHubView.createSubscriptionFormBtn.getAttribute('disabled')).toEqual(null);
+    await operatorHubView.createSubscriptionFormBtn.click();
+    await crudView.isLoaded();
+    await browser.get(`${appHost}/operatorhub/ns/${testName}`);
+    await crudView.isLoaded();
+    await catalogPageView.clickFilterCheckbox('installState-installed');
+    expect(catalogPageView.catalogTileFor('OpenShift Serverless Operator').isDisplayed()).toBe(
+      true,
+    );
+  });
+
+  it(`displays Operator in "Cluster Service Versions" view for "${testName}" namespace`, async () => {
+    await retry(() => catalogPageView.catalogTileFor('OpenShift Serverless Operator').click());
+    await browser
+      .wait(until.presenceOf(installServerlessOperator), 1000)
```

Comment:
Increase the waiting to 15000, it is failing for me.

Reply:
Increased the time to 15000 and also increased the JASMINE_EXTENDED_TIMEOUT_INTERVAL